spec/services/hyrax/repository_fixity_check_service_spec.rb

```diff
@@ -1,9 +1,5 @@
 RSpec.describe Hyrax::RepositoryFixityCheckService do
-  let!(:file) do
-    create(:file_set).tap do |file|
-      file.add_file(File.open(fixture_path + '/world.png'), path: 'content', original_name: 'world.png')
-    end
-  end
+  let!(:file) { create_for_repository(:file_set) }
```

Comment:
Do we not need the attached file? Why?

Reply:
No, it isn't being used actually.  The test is just checking that `FileSetFixityCheckService#fixity_check` is being called:
https://github.com/samvera/hyrax/blob/2d8f81e1be8181644b6a57165a442aec4107b44d/spec/services/hyrax/repository_fixity_check_service_spec.rb#L8-L9